Update GovCloud migration process info for outside-TTS use

brittag opened the pull request:
**Don't merge yet.** This needs to be updated with the upcoming link to self-invitation for GovCloud access.

These changes are meant to help non-18F people migrate their orgs to GovCloud, to support [this message](https://docs.google.com/document/d/1l-akEeE2cRxML_svuQTmo__naN80sN-Jj_n6Jqt9DoE/edit#heading=h.jgcxtjglazl) and related messaging. Feel free to add more improvements to this WIP branch - this is just a start to help.

Changes:

* Start with getting access to GovCloud.
* Provide alternative to org request form for people outside TTS.
* Remove links to incomplete info about service migration.
* Remove "unresolved details" chunk.
* Include the updated text from https://github.com/18F/cg-site/pull/763 to avoid merge conflicts.

Commit:
Updating migration process info

fureigh committed:
Add deadline for migrating apps to GovCloud

https://github.com/18F/cg-product/issues/403

fureigh commented on content/docs/apps/govcloud.md:
```diff
@@ -83,18 +83,15 @@ Any labor involved from your project team is the responsibility of your project.
+1. [Get access to the GovCloud environment] **SELF-INVITATION LINK TBD**.
+1. Request the creation of your org in the GovCloud environment:
+    * If you are in 18F/TTS at GSA, fill out the [organization request form](https://docs.google.com/a/gsa.gov/forms/d/e/1FAIpQLSd4HmcGfJW3EBnpewTFDD-urRFPp1LN0DcwNB_FxZgUn8ho9g/viewform?c=0&w=1). Specify that the org should be created in `GovCloud`. An admin will confirm the information, [create the org]({{< relref "docs/ops/create-org.md#non-sandboxes" >}}) for you, and notify you. This should  ...
+    * For everyone else, one of your org managers should email [cloud-gov-support@gsa.gov](mailto:cloud-gov-support@gsa.gov) to request the creation of your org in GovCloud. Include the name of your East/West org so that we can create the new org with the appropriate org name.
```
Should they also specify the quota they'd like, or does the self-invitation process include that?

fureigh replied:
@jcscottiii @rememberlenny [Self-invitation](https://account.fr.cloud.gov/signup) is now a thing (:tada:)... is requesting a creation of an org still a manual process that requires them to email us the org name and desired quota, though? Asking so I can update this step.

brittag replied:
Yes, creating an org is still a manual process they should email us about. Self-invitation is unrelated - it's just about access to the GovCloud environment.

fureigh committed:
Add self-invitation link

...now that https://github.com/18F/cg-uaa-extras/pull/81 is complete.

https://github.com/18F/cg-site/pull/769

fureigh commented on content/docs/apps/govcloud.md:
```diff
@@ -83,18 +81,15 @@ Any labor involved from your project team is the responsibility of your project.
-    * An admin will confirm the information, and [create the org]({{< relref "docs/ops/create-org.md#non-sandboxes" >}}) for you.
+1. [Get access to the GovCloud environment](https://account.fr.cloud.gov/signup).
+1. Request the creation of your org in the GovCloud environment:
+    * If you are in 18F/TTS at GSA, fill out the [organization request form](https://docs.google.com/a/gsa.gov/forms/d/e/1FAIpQLSd4HmcGfJW3EBnpewTFDD-urRFPp1LN0DcwNB_FxZgUn8ho9g/viewform?c=0&w=1). Specify that the org should be created in `GovCloud`. An admin will confirm the information, [create the org]({{< relref "docs/ops/create-org.md#non-sandboxes" >}}) for you, and notify you. This should  ...
```
@jcscottiii @rememberlenny Are these non-18F/TTS instructions accurate?

jcscottiii replied:
Would like to make sure they tag their subject line with something like `[GovCloud Org Creation]` to ensure we don't lose track. Besides that, seems good

fureigh replied:
@jcscottiii Thanks! Do we need them to specify a quota too or nah?

jcscottiii replied:
nahh

fureigh replied:
I'll move forward on merging this, but: how/when will we get this info from them? Or do they just get set up with a default?

fureigh reviewed (changes requested):
I think this is good to go, pending confirmation that the flagged non-18F/TTS instructions are accurate.

fureigh committed:
Update non-TTS org request instructions

* Add [GovCloud Org Request] subject heading to mailto link
* Request [GovCloud Org Request] subject heading (for manual email)
* Streamline explanation of orgs. ("I heard you like orgs..." :) )

mogul commented on content/docs/apps/govcloud.md:
```diff
@@ -83,18 +81,15 @@ Any labor involved from your project team is the responsibility of your project.
+1. [Get access to the GovCloud environment](https://account.fr.cloud.gov/signup).
+1. Request the creation of your org in the GovCloud environment:
+    * If you are in 18F/TTS at GSA, fill out the [organization request form](https://docs.google.com/a/gsa.gov/forms/d/e/1FAIpQLSd4HmcGfJW3EBnpewTFDD-urRFPp1LN0DcwNB_FxZgUn8ho9g/viewform?c=0&w=1). Specify that the org should be created in `GovCloud`. An admin will confirm the information, [create the org]({{< relref "docs/ops/create-org.md#non-sandboxes" >}}) for you, and notify you. This should  ...
+    * For everyone else, one of your org managers should email [cloud-gov-support@gsa.gov](mailto:cloud-gov-support@gsa.gov?subject=[GovCloud Org Request]) to request the creation of your org in GovCloud. Please include "[GovCloud Org Request]" in the subject heading and specify the name of your current East/West org.
```
You can provide a default body for the `mailto:` URL that prompts them for any necessary information as they fill out the mail.

fureigh replied:
Good call. Seems like there may be further changes to what constitutes the necessary information, so I'll merge this as-is and can start a PR with a minimal version.

fureigh replied:
...ah, this request is blocking the merge. Commit TK, then.

fureigh committed:
Add suggested body for org request email

Also corrects spaces in the subject.
